boards/arm/v2m_musca/doc/v2m_musca.rst

```diff
@@ -0,0 +1,412 @@
+.. _v2m_musca_board:
+
+ARM V2M Musca
+##############
+
+Overview
+********
+
+The v2m_musca board configuration is used by Zephyr applications that run on
+the V2M Musca board. It provides support for the Musca ARM Cortex-M33 CPU and
+the following devices:
+
+- Nested Vectored Interrupt Controller (NVIC)
+- System Tick System Clock (SYSTICK)
+- Cortex-M System Design Kit GPIO
+- Cortex-M System Design Kit UART
+
+.. image:: img/v2m_musca.png
+     :width: 442px
+     :align: center
+     :height: 377px
+     :alt: ARM V2M Musca
+
+More information about the board can be found at the `V2M Musca Website`_.
+
+Hardware
+********
+
+ARM V2M MUSCA provides the following hardware components:
+
+
+
+- ARM Cortex-M33
+- ARM IoT Subsystem for Cortex-M33
+- Memory
+
+  - 128KB SRAM
+  - 2MB of external SRAM
+  - 8MB of external QSPI flash.
+
+- Debug
+
+  - JTAG, SWD & 4 bit TRACE
+  - DAPLink with a virtual UART port
+
+- Arduino interface
+
+  - 16 3V3 GPIO.
+  - UART.
+  - SPI.
+  - I2C.
+  - I2S.
+  - 3-channel PWM.
+  - 6-channel analog interface.
+
+- On-board Peripherals
+
+  - User RGB LED
+  - Gyro sensor
+  - Combined ADC/DAC/temperature sensor
+
+
+User push buttons
+=================
+
+The v2m_musca board provides the following user push buttons:
+• PBON power on/off.
```

Comment:
Use either a ``-`` or ``*`` for a bullet list (not this UTF-8 character).  Also, add a blank line before the first item in the list.